app/src/main/java/org/mozilla/fenix/tabstray/TrayPagerAdapter.kt

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.tabstray
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import org.mozilla.fenix.tabstray.BrowserTabViewHolder.Companion.LAYOUT_ID_NORMAL_TAB
+import org.mozilla.fenix.tabstray.BrowserTabViewHolder.Companion.LAYOUT_ID_PRIVATE_TAB
+import org.mozilla.fenix.tabtray.FenixTabsAdapter
+
+class TrayPagerAdapter(
```

Comment:
Should we be add tests for this? :)

Reply:
Yes! Tests are coming in a follow-up PR. Ran into some local dependency issue with Robolectric for now.

Reply:
👍🏽